docs/end-user-topics/dois.rst

```diff
@@ -69,10 +69,18 @@ Automatic Dockstore DOI Generation
 .. note::
     This feature is currently only available on request. Please contact the Dockstore team at `discuss.dockstore.org <https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506>`__ if you want to participate.
 
-* Automatically mints DOIs for all Git tags pushed to GitHub repositories that have a corresponding published entry in Dockstore
-* The DOIs are minted in the Zenodo Dockstore community
-* No Zenodo account is required to mint DOIs
-* A linked Zenodo account is required to edit DOIs
+Dockstore will automatically mint a DOI for a version that corresponds to a Git tag, if:
+
+* The entry is published on Dockstore
+* The version has at least one author in its metadata
```

Comment:
The version also has to be valid https://github.com/dockstore/dockstore/blob/d8e12f97434dc60437e8e6bfffbec5ac00da798b/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java#L965